docs/solana/migration/hotspot-operator.mdx

```diff
@@ -99,14 +100,16 @@ billion tokens, distributed to all active Hotspots at the time of the Snapshot.
 The migration service will consider a Hotspot as "Active" if the Snapshot includes any of the
 following:
 
-- Witnessed and been valid/invalid witnesses.
-- Beaconed and had valid/invalid witnesses.
-- Transferred data.
+- Witnessed and was a valid witness
+- Beaconed and had valid witnesses
+- Transferred LoRaWAN data and been paid for it
 
 A Hotpsot is not "Active" if:
```

Comment:
I could see someone being confused if this list is "any" of these conditions or "all" of these conditions.